pulp_container/app/registry.py

```diff
@@ -225,3 +228,17 @@ async def get_by_digest(self, request):
                 return await Registry._dispatch(artifact.file, headers)
             else:
                 return await self._stream_content_artifact(request, web.StreamResponse(), ca)
+
+    @staticmethod
+    async def _empty_blob():
+        empty_tar = [
+            31, 139, 8, 0, 0, 9, 110, 136, 0, 255, 98, 24, 5, 163, 96, 20, 140, 88, 0, 8, 0, 0, 255,
+            255, 46, 175, 181, 239, 0, 4, 0, 0,
+        ]
```

Comment:
I don't really like, what black does to these lines. Do you think a `b"\x1f..."` string would look better?